dbtmetabase/metabase.py

```diff
@@ -120,6 +121,7 @@ def __init__(
         user: str,
         password: str,
         verify: Optional[Union[str, bool]] = None,
+        cert: Optional[Union[str, Tuple[str, str]]] = None,
```

Comment:
So it's either a string or a tuple of two strings?

Reply:
Yes, you can verify it in the requests documentation at the Client Side Certificates part https://requests.readthedocs.io/en/latest/user/advanced/